src/test/java/org/jabref/model/jabmap/MindMapClassTest.java

```diff
@@ -0,0 +1,120 @@
+package org.jabref.model.jabmap;
+
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class MindMapClassTest {
+    private static MindMapNode node1;
+    private static MindMapNode node2;
+    private static MindMapEdge edge1;
+
+    // should use local instead
+    // private static final MindMap MYMINDMAP = new MindMap();
+
+    @BeforeAll
+    public static void setup() {
+        node1 = new MindMapNodeBuilder()
+                .withCitationKey("cite1")
+                .withId(1L)
+                .withLabel("node1")
+                .withXPos(50)
+                .withYPos(50)
+                // .withIcons(icons)
+                .build();
+
+        node2 = new MindMapNodeBuilder()
+                .withCitationKey("cite2")
+                .withId(2L)
+                .withLabel("node2")
+                .withXPos(70)
+                .withYPos(70)
+                .build();
+
+        edge1 = new MindMapEdgeBuilder()
+                .withDirection(EdgeDirection.DEFAULT)
+                .withLabel("edge1")
+                .withNode1Id(1L)
+                .withNode2Id(2L)
+                .build();
+    }
+
+    @Test
+    void getNodesTest() throws NoSuchFieldException, IllegalAccessException {
+        // given
```

Comment:
I think the empty lines group enough (in this case). Please remove "given", "when", "then" in the whole class

Reply:
Thanks a lot for the suggestions. Will try to fix these immediately.